src/library_wasmfs.js

```diff
@@ -349,8 +373,16 @@ FS.createPreloadedFile = FS_createPreloadedFile;
       __wasmfs_readdir_finish(state);
       return entries;
     }),
-    // TODO: mount
-    // TODO: unmount
+    mount: (type, opts, mountpoint) => {
+      if (!type) {
+        throw new Error("FS is not valid.");
+      }
```

Comment:
I agree. On a side note, one thing I noticed was that in the legacy JS API trying to pass `NODEFS` without `-lnodefs.js` would give a string message telling the user that `-lnodefs.js` should be used. However in WasmFS, I would directly get a `NODEFS is not defined` error. From what I can tell, `parseTools.js` should be responsible for finding the global uses and replacing them with the string message. Should WasmFS also use this behavior?

Reply:
@kripken do you have thoughts about that -lnodefs.js error message?

Reply:
Yes, those error messages are very useful for people, I think, since "X is not defined" doesn't point the user to the right fix. Hopefully it's a simple matter to get that working in `parseTools`?